apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx

```diff
@@ -175,6 +179,11 @@ export default function ListBoxSearch({
       case 'Escape': {
         focusRow(container);
         cancel();
+        abortListObjectSearch();
```

Comment:
The reason why I'm aborting list search here is because of a recent commit [here](https://github.com/qlik-oss/nebula.js/pull/1531/files#diff-79b554891611ff72399687b538975e8371f2825a6c0968181db2ce309fb58ca3) where a ListboxSearch condition for filterpane has been removed. As far as I know, the condition cannot not be there anymore so instead, Esc key can provide the condition to empty the searchBox, hide it and let the focus back to the main listbox, otherwise the searchbox stays open and pressing Esc or any other key will not help it to go back to its initial state.

Reply:
hmm. maybe consider if the linked change should even apply to listbox popovers used by table.
I don't see any option to cycle a cyclic dimension in it anyway.

Reply:
@T-Wizard sorry couldn't follow you, what do you mean by "if the link change ..." which link?
I'm not familiar with cyclic dimension conditions in this case. I just asked @linhnguyen-qlik and she explained briefly that they don't want the user go to selection mode until clicks an item in the list so the second option came up that is what you see in this code block.

Reply:
> if the link change ..." which link?

the change you linked

> [here](https://github.com/qlik-oss/nebula.js/pull/1531/files#diff-79b554891611ff72399687b538975e8371f2825a6c0968181db2ce309fb58ca3)

also when i tested on https://qcs.us.qlik-stage.com/ I don't see any cycle button in the listbox popover for tables.

Reply:
The cyclic dimension only shows up in filter-pane at the moment. For the table we haven't implemented for cyclic so I don't think we need to change how it works for the list box in table. For me as long as the listbox popover works as described above for popover in filter-pane I am happy.